.env.production

```
@@ -1 +1 @@
-REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
+REACT_APP_API_URL=http://360ApiTrain.gordon.edu/
```

This needs to be https. We shouldn't be pushing changes to this file unless there was a specific, intentional reason to do so.
```suggestion
REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
```

In the future, when adding files to your commit, don't add this file or `setupProxy.js`. That should prevent the hassle of having to revert changes.